src/player.js

```diff
@@ -0,0 +1,57 @@
+class Player {
+  constructor(col, row, img) {
+    this.col = col;
+    this.row = row;
+    this.width = PLAYER_WIDTH;
+    this.height = PLAYER_HEIGHT;
+    this.img = img;
+  }
+
+  preLoad() {
```

Comment:
Careful! JavaScript is case sensitive! `preload` here is written as `preLoad` (uppercase 'L') which will throw an error!

Rename it to `preload` for consistency :v: